PLBenchmarks/tests/test_utils.py

```diff
@@ -0,0 +1,953 @@
+"""
+Unit and regression test for the PLBenchmarks package.
+"""
+
+# Import package, test suite, and other packages as needed
+from PLBenchmarks import utils
+import pytest
+import warnings
+
+
+def test_find_pdb_url():
+    """tests the find_pdb_url function"""
+    pdbs = ["6R8L", "6N5O", "6SVL", "6J5R", "6J6D"]
+    for pdb in pdbs:
+        assert (
+            f"REP1http://www.rcsb.org/structure/{pdb}REP2{pdb}REP3"
+            == utils.find_pdb_url(pdb)
+        )
+
+    strings = [f"REP1http://www.rcsb.org/structure/{pdb}REP2{pdb}REP3" for pdb in pdbs]
+    for string in strings:
+        assert string in utils.find_pdb_url(" ".join(pdbs)).split()
+
+    with pytest.warns(UserWarning):
+        utils.find_pdb_url("fakepdb")
+
+    pdbs[3] = "fakepdb2"
+    with pytest.warns(UserWarning):
+        utils.find_pdb_url(" ".join(pdbs))
+
+
+def test_find_doi_url():
+    assert (
+        "REP1http://dx.doi.org/10.1021/acs.jctc.8b00640REP2Mobley et al., J. Chem. Theory Comput. 2018REP3"
+        == utils.find_doi_url("10.1021/acs.jctc.8b00640")
+    )
+
+    with pytest.warns(UserWarning):
+        assert "fakeDOI" == utils.find_doi_url("fakeDOI")
+
+
+def test_convert_value():
```

Comment:
Consider splitting this test up into its smaller bits (`test_convert_dg`, `test_convert_ki`, `test_convert_ic50`, etc.) so that if something breaks, it's clear which part causes the failure

Reply:
It also looks like some slick use of `pytest.mark.parametrize` could reduce the lines of code, but I'm not sure if the tests are quite homogeneous enough to allow that

Reply:
I split the tests up up. `pytest.mark.parametrize` would be great, will add it as an issue